README.rst

```diff
@@ -62,103 +62,6 @@ Locally
 
 #.  Open a web browser to: http://127.0.0.1:8000
 
-Docker
-~~~~~~
-
-Alternatively, you can build and run the xblock-sdk in Docker (we are using docker compose which
-can be installed as explained at https://docs.docker.com/compose/install/)
-
-After cloning this repository locally, go into the repository directory and build the Docker image::
-
-    $ make docker_build
-
-or manually run
-
-    $ docker compose build
-
-You can then run the locally-built version using the following command::
-
-    $ make dev.up
-
-or manually run::
-
-    $ docker compose up -d
-
-You should now be able to access the XBlock SDK environment in your browser at http://localhost:8000
-
-To stop the container (without removing data) by::
-
-    $ make dev.stop
-
-or manually run::
-
-    $ docker compose stop
-
-Note, using::
-
-    $ make dev.down
-
-or::
-
-    $ docker compose down
-
-will shut down the container and delete non-persistent data.
-
-You can open a bash shell in the running container by using::
-
-    $ make app-shell
-
-or::
-
-    $ docker container exec -it edx.devstack.xblock-sdk bash
-
-Testing
-~~~~~~~
-
-If using Docker, all these commands need to be run inside the xblock-sdk container.
```

Comment:
I think you just want to drop this line but you want to leave the rest of the testing instructions for now.

Reply:
good call